### DISCUSSION TITLE
Profissionais para criação Blog posts sobre Tecnologia

### DISCUSSION THREAD

**Discussion by scarpelini** [frontendbr/forum#984] [Discussão]
Bom dia pessoal, gostaria de saber de vocês da comunidade Front-end BR se interessariam em escrever sobre tecnologia para o Blog de uma empresa privada. Nós estamos buscando desenvolvedores experientes que possuam uma boa gama de conhecimento que possam compartilhar (em inglês de preferência) em nosso Blog.

Como nossa equipe ainda é reduzida, estamos em busca de profissionais que gostam de escrever sobre tecnologia(desenvolvimento Front-end, Back-end, Chatbots, DevOps, Infra, etc) para nos ajudar com essas demandas de criação de conteúdo relevante, onde além de nos ajudar em quesitos de posicionamento em mecanismos de busca, também estaremos compartilhando conhecimento que é um de nossos objetivos.

Alguém teria interesse? 
Se sim, qual tema possui knowhow para abordar?

A ideia é conversar pontualmente com cada um para chegar num acordo bacana e que funcione bem pra ambas as partes.

Meu e-mail é julio@santa.io para caso queiram entrar em contato diretamente.

Abraços

**Comment by woliveiras** [comment]
@scarpelini, não entendi muito bem... A issue está meio estranha.

Sua empresa é de desenvolvimento de software, possui cultura de compartilhamento de conhecimento, mas estão buscando pessoas de fora para escrever para o blog da empresa pra ter conteúdo?

Ficou parecendo que estão querendo ter um monte de artigo para mostrar para os seus clientes que vocês tem conhecimento, mas na realidade é um conhecimento comprado de fora e não de dentro 😱😱😱 (não me entenda mal, é só o que estou interpretando depois de acessar o site de vocês e feito o match com a descrição da issue, viu... não quero lhe ofender).

Poderia nos informar qual a intenção de vocês com o conteúdo, mano?

Porque, se for pra ajudar a comunidade, de fato, seria melhor postarmos em blogs da própria comunidade, como Tableless, iMasters, BrazilJS e fortalecer esses portais, não?

**Comment by scarpelini** [comment]
Fala @woliveiras sem problemas, fique tranquilo hehe

Você não entendeu errado não. A intenção é criar mais conteúdos do que nós conseguimos comportar com nossa equipe interna(que é reduzida). 

Por conta do nosso alto fluxo de trabalho, estamos em busca de desenvolvedores parceiros(não apenas para criação desses conteúdos) para que possamos criar esse material que nos indexe bem nos mecanismos de busca. O fim basicamente é para SEO num primeiro momento, e também por consequência o compartilhamento de informações. Nós planejaríamos os temas juntos com os profissionais para que sejam interessantes para ambas as finalidades.

E sim, é um "conhecimento comprado", porém que reflete o nosso conhecimento interno, por isso o planejamento conjunto. A ideia não é sair escrevendo sobre qualquer coisa envolvendo especificamente tecnologia. 

Um exemplo de assuntos que pretendemos abordar ReactJS, React Native, Messenger Bots e  Wordpress em termos de tecnologias. Planajemento, Estratégia e Business Development em termos de práticas envolvendo uma empresa de desenvolvimento de software. Benefícios da Metodologia SCRUM como meio de um processo de desenvolvimento contínuo com progresso transparente.

Essas são alguns assuntos que fazem parte da nossa realidade diária.
Melhor explicado? Continua tão zuado quanto pareceu, mais ou menos? 

Abraços
